trust_path/utilities/add_indexes.php

```
@@ -0,0 +1,68 @@
+ * 2. Run: php add_indexes.php
+ *
+ * @copyright   Simon Wilkinson 2024+ (https://tuskfish.biz)
+ * @license     https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html GNU General Public License (GPL) V2
+ * @author      Simon Wilkinson <simon@isengard.biz>
+ * @since       2.0
+ * @package     database
+ */
+
+// CONFIG: Uncomment line below and SET THE CORRECT PATH to mainfile.php.
+// require_once '../../mainfile.php';
+require_once TFISH_PATH . 'header.php';
```

PHPStan (2.2.1)

[error] Path in require_once() "libraries/tuskfish/header.php" is not a file or it does not exist.


(requireOnce.fileNotFound)